### PR TITLE
Kibana ami

### PR DESCRIPTION
**Pull request by Cloudyeli**

### CODE REVIEW

File: ec2_kibana.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
+resource "aws_instance" "kibana_server" {
+  ami = data.aws_ami.kibana_on_ubuntu.id
+  instance_type = "t3.medium"
+  subnet_id = data.aws_subnet.public_subnet.id
+  root_block_device  {
+        #device_name = "elasticSearch-ebs"
+        volume_size = "50"
+    }
+user_data = templatefile("${path.module}/user-data.sh.tpl",
```
Comment (pasc-ed) [inline ec2_kibana.tf]: identation is not good, could you run a `terraform fmt` please

**Review by pasc-ed — CHANGES_REQUESTED** [review]
just some fmt required.

**Review by pasc-ed — APPROVED** [review]
LGTM

**Review by KlToti — APPROVED** [review]
gut gemacht!

**Pull request merged**